src/test/python_tests/test_code_actions.py

```diff
@@ -14,6 +15,7 @@
 TEST_FILE_PATH = constants.TEST_DATA / "sample1" / "sample.py"
 TEST_FILE_URI = utils.as_uri(str(TEST_FILE_PATH))
 LINTER = utils.get_server_info_defaults()["name"]
+TIMEOUT = 10  # 10 seconds
```

Comment:
is this in seconds or milliseconds?

Reply:
It is seconds. Python in most cases uses seconds and floating point for milliseconds